modules/sdk-coin-baby/src/lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const validDenoms = ['baby', 'tbaby', 'ubbn'];
+export const accountAddressRegex = /^(bbn)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l]{38})$/;
+export const validatorAddressRegex = /^(bbnvaloper)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l]{38})$/;
+export const contractAddressRegex = /^(bbn)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l]+)$/;
```

Comment:
This regex doesn't seems to be correct. Ideally it should accept all alphanumeric characters in the address body of length 38.